src/test/java/uk/gov/companieshouse/acsp/service/FilingServiceTest.java

```diff
@@ -103,26 +99,30 @@ void init() {
     private void setACSPDataDto() {
         acspDataDto = new AcspDataDto();
         acspDataDto.setId(ACSP_ID);
-        acspDataDto.setFirstName(FIRST_NAME);
-        acspDataDto.setLastName(LAST_NAME);
+
+        if (acspDataDto.getApplicantDetails() == null) {
```

Comment:
We know its null at this point... no need to check for null... you can just create the new applicantdetails everytime

Reply:
I have removed. Thanks